Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rails:4.2.1
+VOLUME ["/usr/src/app"]
+WORKDIR /usr/src/app
+ADD Gemfile /usr/src/app/Gemfile
```

Comment:
Quick question: does Gemfile.lock is auto-generated or something? Because if it changes often and it's out of our control it can make the build process take longer by invalidating the Dockerfile cache.

I'm just asking, I have no idea.